e2e/packages/contracts/src/codegen/tables/Multi.sol

```diff
@@ -46,25 +46,15 @@ library Multi {
    * @return _keySchema The key schema for the table.
    */
   function getKeySchema() internal pure returns (Schema) {
-    SchemaType[] memory _keySchema = new SchemaType[](4);
-    _keySchema[0] = SchemaType.UINT32;
-    _keySchema[1] = SchemaType.BOOL;
-    _keySchema[2] = SchemaType.UINT256;
-    _keySchema[3] = SchemaType.INT120;
-
-    return SchemaLib.encode(_keySchema);
+    return Schema.wrap(0x0034040003601f2e000000000000000000000000000000000000000000000000);
```

Comment:
Neat! Is there any gas impact to declaring these as constants, ie. `Schema constant KEY_SCHEMA = ...`?

Wondering if we can go even further and remove the functions (`getKeySchema`), leaving only the constants. Seems more fitting if all the functions do is return a hardcoded value.

Reply:
yep, that'd be the next step (see OP) but wanted to wait for the other PR to land first